src/draw.c

```diff
@@ -29,11 +29,16 @@ struct color {
         double a;
 };
 
+struct gradient {
+        struct color *cs;
```

Comment:
I'd suggest to use `colors` here as it is easier to follow on the usage side.

Reply:
If you mean the variable name, sure